doc/source/tutorial/stats/continuous_genhyperbolic.rst

```diff
@@ -0,0 +1,139 @@
+
+.. _continuous-genhyperbolic:
+
+Generalized Hyperbolic Distribution
+===================================
+
+The Generalized Hyperbolic Distribution is defined as the normal variance-mean mixture with Generalized Inverse Gaussian distribution as the mixing distribution.
+The "hyperbolic" characterization refers to the fact that the shape of the log-probability distribution can be described as a hyperbola. Hyperbolic distributions are sometime referred to as semi-fat tailed because their probability density decrease slower than "sub-hyperbolic" distributions (e.g. normal distribution, whose log-probability decreases quadratically), but faster than other "extreme value" distributions (e.g. `pareto` distribution, whose log-probability decreases logarithmically).
+
+Functions
+---------
+
+Different parameterization exists in the literature; SciPy implements the location-scale as in Prause (1999).
+
+.. math::
+   :nowrap:
+
+    \begin{eqnarray*}
```

Comment:
For the record, I copied down this PDF, did the substitutions, and got the original parameterization as written in  Barndorff .